docs/testcases/UI/System_Management/3059-3075_audit_pipe_mount_operation/3059_2.md

```diff
@@ -0,0 +1,28 @@
+# Data access audit (operations with folders)
+
+Test verifies that any access via pipe CLI to the data stored in the object storages is logged.
+
+**Prerequisites**:
+- Admin user
+- Non-admin user
+- perform [3059_1](3059_1.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the non-admin user from the prerequisites | |
+| 2 | At the **Runs** page, click the run launched at step 4 of [3059_1](3059_1.md) case | |
+| 3 | Wait until the **SSH** hyperlink appears | |
+| 4 | Click the **SSH** hyperlink | |
+| 5 | In the opened tab, enter and perform the `pipe storage cp -r storage1/folder1 storage2/folder1` command | |
+| 6 | Logout and login as the admin user from the prerequisites | |
+| 7 | Open the **Settings** page | |
+| 8 | Open the **SYSTEM MANAGEMENT** page | |
+| 9 | Select the **LOGS** tab | |
+| 10 | Click ***Show advanced*** button | |
+| 11 | Click **Type** field and select `audit` value from drop-down | For each files from folder1 (*inner_file1* and *inner_file2*) records <br>`READ storage1/folder1/<file>` <br> `WRITE storage2/folder1/<file>`  <br> are shown in the top of logs records |
+| 12 | Logout | |
+| 13 | Repeat steps 1-12 for `pipe storage -r mv storage1/folder2 storage2/folder2` command | For each files from folder2 (*inner_file3* and *inner_file4*) records <br> `READ storage1/folder2/<file>` <br> `WRITE storage2/folder2/<file>` <br> `DELETE storage1/folder2/<file>` <br> are shown in the top of logs records |
+| 14 | Repeat steps 1-12 for `pipe storage rm -r storage1/folder1 -y` command | For each files from folder1 (*inner_file1* and *inner_file2*) record <br> `DELETE storage1/folder1/<file>` <br> is shown in the top of logs records |
+
+**After:**
```

Comment:
I suggest to remove the section **After**, as this run is still need in the case 3075

Reply:
Done